server/src/main/java/org/opensearch/extensions/ExtensionActionListener.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions;
+
+import java.util.ArrayList;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.action.ActionListener;
+import org.opensearch.action.admin.indices.analyze.AnalyzeAction.Response;
+
+/**
+ * ActionListener = for ExtensionsOrchestratore
```

Comment:
(Typo) extra e at the end. Wouldn't hold up merging for this but since I've requested other changes...

Reply:
I fixed this in my next PR that will be up once this one merges.